src/overlays/gamestates/ovl_file_choose/z_file_nameset.c

```diff
@@ -1747,13 +1747,21 @@ void FileSelect_DrawOptionsImpl(GameState* thisx) {
             gDPSetEnvColor(POLY_OPA_DISP++, 0, 0, 0, 255);
         }
 
+#ifndef AVOID_UB
         //! @bug Mistakenly using sOptionsMenuHeaders instead of sOptionsMenuSettings for the height.
         //! This is also an OOB read that happens to access the height of the first two elements in
         //! sOptionsMenuSettings, and since all heights are 16, it works out anyway.
+#define sOptionsMenuSettingsBug sOptionsMenuHeaders
+#else
+        // Avoid UB: Use the correct array for the heights to avoid reading out of bounds memory that may not
+        // happen to work out nicely.
+#define sOptionsMenuSettingsBug sOptionsMenuSettings
```

Comment:
there's another case of this for PAL versions, is that also an OOB read?

Reply:
The other case only loops up to 4 so remains in bounds throughout, so there's no need to avoid UB there

Reply:
It seems like `i` can go up to either 6-1 or 8-1, depending on if the language is German or not, but either way would read OOB I think

Reply:
Oh I was looking in the wrong place since it didn't have a bug comment lol. Done

Reply:
ah sorry. I wonder if we can combine the two and define `sOptionsMenuSettingsBug` only once overall instead of once in each case, but it's not important